.gitmodules

```diff
@@ -46,3 +46,6 @@
 [submodule "src/watchmaker/static/salt/content"]
 	path = src/watchmaker/static/salt/content
 	url = https://github.com/plus3it/watchmaker-salt-content.git
+[submodule "src/watchmaker/static/salt/formulas/fup-formula"]
+	path = src/watchmaker/static/salt/formulas/fup-formula
+	url = git@github.com:plus3it/fup-formula.git
```

Comment:
eh, let's use https remotes, like all the rest of the submodules...

Reply:
Oop. Sorry. Habit (and wasn't specified in the doc 😛)

Fixing, shortly.